src/components/seekbar/seekbar.js

```diff
@@ -135,6 +135,7 @@ class SeekBar extends Component {
       return;
     }
     e.preventDefault(); // fixes firefox mouseup not firing after dragging the scrubber
+    e.stopPropagation(); // prevent dragging while visibility plugin registered and configured to floating
```

Comment:
The in-line comment is a bit not relevant as this is a core library but comment mentions an optional plugin which is a third party. Consider making a more common or general comment here.